cloud-common/src/main/resources/application.yml

```diff
@@ -159,6 +159,12 @@ cb:
   blueprint:
     cm:
       defaults: >
+                CDP 1.2 - Data Engineering: Apache Spark, Apache Hive, Apache Oozie=cdp-data-engineering-702;
+                CDP 1.2 - Data Mart: Apache Impala, Hue=cdp-data-mart-702;
+                CDP 1.2 - Operations Data Mart: Apache Impala, Hue, Apache Kudu, Apache Spark=cdp-op-data-mart-702;
+                CDP 1.2 - Operational Database: Apache HBase=cdp-opdb-702;
+                CDP 1.2 - SDX Light Duty: Apache Hive Metastore, Apache Ranger, Apache Atlas=cdp-sdx-702;
```

Comment:
Can you please rebase the PR. I've merged a change that adds SDX blueprints for CDP 1.2. I'm not sure if you have made also a change there or just a copy with a new name.

Reply:
Done.